src/utils/ResponseHandler.js

```diff
@@ -18,7 +18,7 @@ export default class ResponseHandler {
    * @returns {void}
    */
   static success(res, data, status = 200) {
-    res.status(status).json({
+    return res.status(status).json({
```

Comment:
I believe i have implemented this method before. why the repetition?

Reply:
Please delete this file. All response helper methods are already declared in the helpermethod.js file